trestle/core/commands/init.py

```diff
@@ -70,8 +75,14 @@ def _create_directories(self, root: pathlib.Path) -> None:
             # Create directories
             for directory in directory_list:
                 directory.mkdir(parents=True, exist_ok=True)
-                file_path = pathlib.Path(directory) / const.TRESTLE_KEEP_FILE
-                file_path.touch()
+                if fs.is_windows():
+                    file_path = pathlib.Path(directory) / const.TRESTLE_KEEP_FILE_WINDOWS
+                    file_path.touch()
+                    atts = win32api.GetFileAttributes(str(file_path))
+                    win32api.SetFileAttributes(str(file_path), win32con.FILE_ATTRIBUTE_HIDDEN | atts)
+                else:
```

Comment:
Hmm - I just realized you wanted to use 'keep' on windows and not '.keep'.  It's actually fine to use .keep so I would just use the single const.TRESTLE_KEEP_FILE.  So all this code should reduce to simply `make_hidden_file(path/.keep)`. And it will do all the OS specific stuff.  The code should never have to call win32api outside of a few dedicated utility functions.